Makefile

```diff
@@ -208,3 +213,21 @@ test-e2e: ## Runs e2e tests.
 .PHONY: test-e2e
 test-e2e: deploy
 	go test -timeout 20m -count 1 -v -tags e2e -run "$(TEST)" ./test/e2e
+
+# OLM
+.PHONY: bundle
+bundle: manifests
+	kustomize build config/default | operator-sdk generate bundle -q
```

Comment:
I was thinking about bringing `operator-sdk` to external-dns-operator's repository too as I used it for the scaffolding. It's indeed may be useful to be able to update the bundle from Kustomize files but I was wondering about 2 things:
- wouldn't this be misleading to add `operator-sdk` to a `kubebuilder` project
- wouldn't `operator-sdk generate` override a lot of things which we set differently (`ClusterServiceVersion` file) or don't use (scorecard)

Reply:
Do be honest I'm not sure. I'll do a bit more digging to get a better understanding

Reply:
> wouldn't this be misleading to add operator-sdk to a kubebuilder project

Not necessarily because `kubebuilder` is still a dependency for `operator-sdk`
https://github.com/operator-framework/operator-sdk/blob/05d58e209d36f03de92d19fe4dd580a1c784c608/go.mod#L43

Reply:
Yes, operatorSDK depends on kubebuilder but the upstream is a kubebuilder project. I don't think that mentioning an overlapping tech upstream would be much appreciated.
However the whole PR is now well suited for the upstream, so I'm proposing to just change the commit to `carry patch` one and keep operatorSDK.